rust-runtime/aws-smithy-http-server-python/README.md

```diff
@@ -0,0 +1,7 @@
+# aws-smithy-http-server-python
+
+Server libraries for smithy-rs generated servers, targeting pure Python business logic.
+
+<!-- anchor_start:footer -->
+This crate is part of the [AWS SDK for Rust](https://awslabs.github.io/aws-sdk-rust/) and the [smithy-rs](https://github.com/awslabs/smithy-rs) code generator. In most cases, it should not be used directly.
```

Comment:
Not part of the AWS Rust SDK. We have the same phrase in `aws-smithy-http-server`.